manual/contents/src/terminology.asciidoc

```diff
@@ -183,7 +183,7 @@ When used in `WHERE`, `NULL` is treated as `FALSE`.
 
 [[term-projection]]projection::
 an operation taking <<term-result-row, result rows>> as both input and output data.
-This may be a subset of the <<term-identifier, identifiers>> provided in the input, a calculation based on single or multiple identifiers in the input, or both.
+This may be a subset of the <<term-variable, variables>> provided in the input, a calculation based on single or multiple variables in the input, or both.
```

Comment:
... a calculation based on _a_ single or multiple variables in the input, or both.

Adding the _a_? Maybe expanding to _a single variable_? Sounds wrong without it.